CIME/scripts/create_test.py

```diff
@@ -351,14 +351,29 @@ def parse_command_line(args, description):
         config, "ALLOW_BASELINE_OVERWRITE", False, check_main=False
     )
 
+    default = get_default_setting(
+        config, "ALLOW_BASELINE_SKIP", False, check_main=False
+    )
+
     parser.add_argument(
         "-o",
         "--allow-baseline-overwrite",
         action="store_true",
         default=default,
         help="If the --generate option is given, then an attempt to overwrite "
         "\nan existing baseline directory will raise an error. WARNING: Specifying this "
-        "\noption will allow existing baseline directories to be silently overwritten.",
+        "\noption will allow existing baseline directories to be silently overwritten. "
+        "\nIncompatible with --allow-baseline-skip.",
+    )
+
+    parser.add_argument(
+        "--allow-baseline-skip",
+        action="store_true",
+        default=default,
+        help="If the --generate option is given, then an attempt to overwrite "
```

Comment:
Would it be possible to use an [exclusive group](https://docs.python.org/3/library/argparse.html#mutual-exclusion) to have `argparse` automatically only allow one?

Reply:
Good point! Done with 0aa6f5468.